styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/FlyteRunnerHandler.java

```diff
@@ -47,35 +47,39 @@ public FlyteRunnerHandler() {
   public void safeTransitionInto(final RunState state, final EventRouter eventRouter) {
     switch (state.state()) {
       case SUBMITTING:
-        LOG.info("Start flyte exec");
+        LOG.info("Entered state SUBMITTING for: " + state.workflowInstance());
```

Comment:
WorkflowInstance is a complex object, right? I think that we should probably get the workflowId from that

Reply:
I think it has a toString method which concatenates, let me check

Reply:
It should concatenate workflowId and partition

Reply:
👍